aten/src/ATen/cuda/tunable/Tunable.cpp

```diff
@@ -412,14 +432,28 @@ int TuningContext::GetMaxWarmupIterations() const {
   return max_warmup_iterations_;
 }
 
-void TuningContext::EnableTunableOpAndTuning() {
-  EnableTunableOp();
-  EnableTuning();
+void TuningContext::EnableICacheFlush(bool value) {
+  icache_flush_ = value;
+}
+
+bool TuningContext::IsICacheFlushEnabled() const {
+  static const char *env = std::getenv("PYTORCH_TUNABLEOP_ICACHE_FLUSH_ENABLED");
+  if (env != nullptr && strcmp(env, "1") == 0) {
+    return true;
+  }
+  return icache_flush_;
+}
+
+void TuningContext::SetRotatingBufferSize(int size) {
+  rotating_buffer_size_ = size;
 }
 
-void TuningContext::DisableTunableOpAndTuning() {
-  DisableTunableOp();
-  DisableTuning();
+int TuningContext::GetRotatingBufferSize() const {
+  static const char *env = std::getenv("PYTORCH_TUNABLEOP_ROTATING_BUFFER_SIZE");
```

Comment:
If `env` is not set, does it make sense to set the default value to be something like L2 cache size?

Reply:
If env is not set, we don't want this feature to be on be default.  Defaulting to 0 turns this feature off.